src/Psl/DateTime/DateTimeConvenienceMethodsTrait.php

```diff
@@ -7,6 +7,9 @@
 use Psl\Locale\Locale;
 use Psl\Math;
 
+use function intdiv;
+use function min;
+
```

Comment:
should use `Math\minva` and `Math\div` instead.

Reply:
added `minva`

However `div` throws:

>  * throws Exception\ArithmeticException If the $numerator is Math\INT64_MIN and the $denominator is -1.
> * throws Exception\DivisionByZeroException If the $denominator is 0.

Which is not true, cause de denominator is always 12. 
Because I don't want to be leaking those exceptions on these methods (cause they don't make sense), I just used `\intdiv` directly.

Reply:
@psalm-supress is your friend :P